lib/stripe/subscriptions.ex

```diff
@@ -91,6 +91,19 @@ defmodule Stripe.Subscriptions do
 
   ## Example
   ```
+  Stripe.Subscriptions.change "customer_id", "subscription_id", [plan: "plan_id"]
+  ```
+  """
+  def change(customer_id, sub_id, opts) when is_list(opts) do
```

Comment:
Let's be sure we don't break the API :). With this guard in place the existing users who have this library will be mad at me because I broke their code :). So... we need an additional function head here that allows for a key *and* opts. Also, let's be specific with the `opts` - we don't want to take in any old list - let's pattern match as best we can.